dcpy/connectors/edm/recipes.py

```diff
@@ -130,7 +155,13 @@ def import_dataset(
         )
         if preprocessor is not None:
             df = preprocessor(ds.name, df)
+        columns = {
+            column: column.replace("_", "_").replace("'", "_") for column in df.columns
+        }
+        df.rename(columns=columns, inplace=True)
```

Comment:
seems worth noting why this is now needed

Reply:
Oops, there's also a typo there. Replacing underscore with another underscore.